Source/WebKit/UIProcess/API/Swift/WebPage.swift

```diff
@@ -120,20 +125,125 @@ public class WebPage_v0 {
             createObservation(for: \.isLoading, backedBy: \.isLoading),
             createObservation(for: \.serverTrust, backedBy: \.serverTrust),
             createObservation(for: \.hasOnlySecureContent, backedBy: \.hasOnlySecureContent),
+            createObservation(for: \.isWritingToolsActive, backedBy: \.isWritingToolsActive),
             createObservation(for: \.themeColor, backedBy: \.themeColor),
         ]
     }
 
+    public static func handlesURLScheme(_ scheme: String) -> Bool {
```

Comment:
`static` should probably go at the top of the type, like we would for a class method in ObjC.